static/app/views/settings/organizationTeams/allTeamsRow.tsx

```diff
@@ -211,7 +236,8 @@ class AllTeamsRow extends Component<Props, State> {
             display
           )}
         </div>
-        <div>{this.getTeamRoleName()}</div>
+        <div style={{display: roleDisplay()}}>{teamOrgRole}</div>
+        <div style={{display: roleDisplay()}}>{this.getTeamRoleName()}</div>
```

Comment:
Our frontend convention disallow in-line styles in favour of Emotion/Styled Components, so you will want to do something like

```js
<DisplayRoleThing myProp={...}>{teamOrgRole}</DisplayRoleThing>
<DisplayRoleThing myProp={...}>{this.getTeamRoleName()}</DisplayRoleThing>

const DisplayRoleThing = styled(div)<{myProp: boolean}>`
  display: ${prop.myProp === null ? 'none' : 'block'};
`;
```